netbox/extras/choices.py

```diff
@@ -117,10 +117,12 @@ class BookmarkOrderingChoices(ChoiceSet):
 
     ORDERING_NEWEST = '-created'
     ORDERING_OLDEST = 'created'
+    ORDERING_NAME_AZ = 'name'
 
     CHOICES = (
         (ORDERING_NEWEST, _('Newest')),
         (ORDERING_OLDEST, _('Oldest')),
+        (ORDERING_NAME_AZ, _('Name (A-Z)')),
```

Comment:
This ordering isn't technically by name, as not all model have a `name` attribute (circuits, for instance). Let's change this to "alphabetical" to be more accurate.